config/deploy.rb

```diff
@@ -79,6 +79,7 @@
   SSLCertificateFile #{cert_path}/#{cert_host_name}_cert.cer
   SSLCertificateChainFile #{cert_path}/#{cert_host_name}_interm.cer
   SSLCertificateKeyFile #{cert_path}/#{cert_host_name}_key.cer
+  SSLProtocol all -SSLv2 -SSLv3
```

Comment:
Good job figuring this out. Better security is always great and especially if it's something that informs us for the future too.

Reply:
Thanks. Yea, we had this turned off in other configs, but they didn't matter.